SDWebImage/SDWebImageDownloaderOperation.h

```diff
@@ -81,16 +89,6 @@ FOUNDATION_EXPORT NSString * _Nonnull const SDWebImageDownloadFinishNotification
  */
 @property (copy, nonatomic, readonly, nullable) SDWebImageContext *context;
 
-/**
- * The expected size of data.
- */
-@property (assign, nonatomic, readonly) NSInteger expectedSize;
```

Comment:
This one `expectedSize` can be grab from `response` easily. It's no need to provide a seperate property for this.
And also, `NSInteger` does not fully cover the `long long` value from `response. expectedContentLength` and currentlly it does not have ability to identify `NSURLSessionTransferSizeUnknown`(-1)....We should put it as private usage only